src/GeoTables.jl

```diff
@@ -54,9 +54,16 @@ appropriate format based on the file extension.
 
 - `*.geojson` via GeoJSON.jl
 """
-function save(fname, geotable)
-  if endswith(fname, ".geojson")
-    GJS.write(fname, geotable)
+function save(fname, geotable; kwargs...)
+  if endswith(fname, ".shp")
+    geoms = domain(geotable)
+    if !(geoms[1] isa Multi) && !(geoms[1] isa Point)
+      newgeoms = GeometrySet([Multi([geom]) for geom in geoms])
+      geotable = meshdata(newgeoms, etable = values(geotable))
+    end
```

Comment:
@ErickChacon I find it strange that we are fixing this issue in GeoTables.jl, shouldn't we contact the Shapefile.jl maintainers to fix this behavior over there? People will certainly try to save non-Multi geometries with Shapefile.jl and will get the same error right?

Reply:
Yes, that will also happen with other packages trying to use `Shapefile.jl`. I will start an issue to see if they can and want to handle that.